packages/app/obojobo-express/__tests__/models/guest_user.test.js

```diff
@@ -43,11 +41,11 @@ describe('guest user model', () => {
 		const GuestUser = oboRequire('models/guest_user')
 		const g = new GuestUser()
 
-		expect(g.canViewEditor).toBe(undefined)
-		expect(g.canEditDrafts).toBe(undefined)
-		expect(g.canDeleteDrafts).toBe(undefined)
-		expect(g.canCreateDrafts).toBe(undefined)
-		expect(g.canViewDrafts).toBe(undefined)
+		expect(g.canViewEditor).toBe(false)
```

Comment:
These changed because the tests changed how they load config data, but I think it's a little more clear now.